config/locales/referee.yml

```diff
@@ -1,7 +1,21 @@
 en:
   reference_form:
     confirm: Submit reference
-  contact_form:
+  questionnaire_form:
+    experience: Please rate your experience of giving a reference
+    why_that_rating: Tell us why you chose that rating
+    very_poor: Very poor
+    poor: Poor
+    ok: Ok
+    good: Good
+    very_good: Very Good
+    safe_to_work_with_children: If we asked whether a candidate was safe to work with children, would you feel able to answer?
+    is_safe_to_work_with_children: Yes, i could answer
```

Comment:
```suggestion
    is_safe_to_work_with_children: Yes, I could answer
```

Reply:
Never mind just saw Will's comment (https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1297#discussion_r375934727) 👀